src/api/structure/interfaces.ts

```diff
@@ -0,0 +1,30 @@
+export interface Metadata {
+    [key: string]: string;
+}
+
+export interface TestFile {
+    collectedTests: Test[];
+    currentFixture: Fixture;
+}
+
+export interface Fixture {
+    name: string;
+    path: string;
+    testFile: TestFile;
+    meta: Metadata;
+    afterFn: Function | null;
+    beforeFn: Function | null;
+    afterEachFn: Function | null;
+    beforeEachFn: Function | null;
+}
+
+export interface Test {
+    only: boolean;
```

Comment:
Currently, this interface contains only properties used by the TypeScript code. The `only` property is used here: 

https://github.com/DevExpress/testcafe/blob/master/src/runner/bootstrapper.ts#L245

I would like to avoid describing unused properties here and now due to the following reasons:

1. Adding them now won't improve typechecking since TypeScript doesn't check interfaces in the JS code.
2. It's harder to correctly write an interface when you don't have the code that uses it. Likely, an another person will have to fix the interface after another piece of TestCafe code will be rewritten in TypeScript. Please note, that this `Test` describes not the end-user `test` API but our internal structure:

https://github.com/DevExpress/testcafe/blob/master/src/api/structure/test.js#L12

Reply:
ok, totally agree